MonoidalCategories/gap/BraidedMonoidalCategoriesTest.gi

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# MonoidalCategories: Monoidal and monoidal (co)closed categories
+#
+# Implementations
+#
+
+InstallGlobalFunction( "BraidedMonoidalCategoriesTest",
+    
+    function( cat, a, b )
+        
+        local opposite, verbose,
+              
+              a_op, braiding_a_b, braiding_a_b_op, braiding_inverse_a_b, braiding_inverse_a_b_op, 
+              b_op, braiding_b_a, braiding_b_a_op, braiding_inverse_b_a, braiding_inverse_b_a_op;
+        
+        opposite := Opposite( cat );;
+        
+        a_op := Opposite( a );;
+        b_op := Opposite( b );;
+        
+        verbose := ValueOption( "verbose" ) = true;;
+        
+        if CanCompute( cat, "Braiding" ) then
+            
+            if verbose then
+                
+                Display( "Testing 'Braiding' ..." );
```

Comment:
You can put a comment `# COVERAGE_IGNORE_NEXT_LINE` above lines which solely exist for debugging purposes to ignore them when gathering code coverage.

Reply:
Oh, that's nice. Added it.